docs/configuration.rst

```diff
@@ -1,45 +1,66 @@
 Configuration
 =============
 
-The default settings should be enough for most of the cases.
-Although, if you have special requirements this section may help you tune up a little the extension to make it work as you need.
+The default settings cover the most common RTD use case.
```

Comment:
I would not mention RTD here since the extension should work without any change in other places as well. Although, if you think it's necessary to mention it (since it's tied to the following example), "RTD" should be "Read the Docs".

I'm not sold on one or the other, but I want to avoid people not using the extension because they think that it's Read the Docs only. I added this paragraph in the index trying to mitigate this:

> This extension was originally developed to be used on Read the Docs but it can be used in other hosting services as well.

But you probably knows more than me about writing docs :smile: , so I'm open to suggestions.

Reply:
Sounds good. I updated the language - let me know what you think. (I also added Description: and Notes: to the individual config value listings where appropriate, this makes the file a sort of template for future entries.)